qsa-calculator-domain/src/main/java/com/unideb/qsa/calculator/domain/SystemFeature.java

```diff
@@ -4,114 +4,33 @@
  * System features for mapping the values.
  */
 public enum SystemFeature {
-    a,
     Alpha,
-    BcRo,
-    Beta,
-    bN,
     c,
     m,
-    C1,
-    C2,
-    C2S,
-    C2T,
-    C5,
-    C6,
-    CS2,
-    D,
     Da,
     Ds,
-    D2N,
-    D2Q,
-    D2QQ0,
-    D2T,
-    D2W,
-    D2WW0,
-    E0,
-    EQQ0,
     eS,
-    eS1,
-    eS2,
-    eS3,
-    eS12,
-    eS13,
-    eS21,
-    eS22,
-    eS23,
-    eS32,
-    eS33,
-    ESn,
+    eSc1,
+    eSc2,
     eSPow2,
+    eSPow2c1,
+    eSPow2c2,
     eSPow3,
-    ET2,
```

Comment:
Does this mean we don't need them anymore? That is great! If I get this correctly, then these are the outputs. All of them are calculated and not needed to resolve. Thank you!

Reply:
Yes, I removed everything but the input features. Only the ones used as inputs are needed (when getting their corresponding values from a map).